frontend/src/metabase/icon_paths.js

```diff
@@ -440,6 +440,8 @@ export function loadIcon(name: string) {
       width: 16,
       height: 16,
       fill: "currentcolor",
+      role: "img",
+      ariaLabel: name + " icon",
```

Comment:
Would switching this to `"aria-label": ...` avoid explicitly setting the prop in Icon.jsx?